### PR TITLE
Payment Type Controller & Test

### PR DESCRIPTION
**Pull request by nikx1015**
# Description

Created payment type controller which allows the user to create, edit, and delete payment types. I also created a test to make sure the controller is allowing the user to get a single payment type, get all payment types, create a payment type, edit a payment type, and delete a payment type.

## Related Ticket(s)
Ticket 3
Allow developers to access payment type resource

## Steps to Test
git add and git commit what you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout na-paymenttype to access my branch.
If you don't see the most recent version of their files, try git pull origin na-paymenttype
Run your server on their branch and make sure everything looks okay
Once server is running, use your browser to navigate to api/products to ensure that correct information is displayed.

Afterwards, in Visual Studio, go to the TestProduct.cs file under TestBangazonAPI and right-click on it and click on Run Tests. Ensure that all tests come back green.

Check verbiage and comments to make sure they are legible and understandable.

# Checklist:
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by BatsGoScreechh** [comment]
Everything looks good to me. Code is well commented. Tests came back all green and running the server returned the expected result! :+1:

**Comment by ColonelSped** [comment]
Please add the information related to said Ticket.

File: BangazonAPI/TestBangazonAPI/TestBangazonAPI.csproj [hunk 1]
```diff
@@ -12,8 +12,13 @@
...
+    <PackageReference Include="System.Data.SqlClient" Version="4.6.1" />
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\BangazonAPI\BangazonAPI.csproj" />
+  </ItemGroup>
+
 </Project>
```
Comment (ColonelSped) [inline BangazonAPI/TestBangazonAPI/TestBangazonAPI.csproj]: Overall code looks good!

**Review by ColonelSped — CHANGES_REQUESTED** [review]

**Comment by ColonelSped** [comment]
Please use this as a model for your "Steps to Test"

git add and git commit whatever you're working on on your branch
git checkout master to switch to your local master branch
git fetch --all will give you access to your teammate's branches (if they've pushed them up)
git checkout <your-teammate's-branch-name> to switch to their branch
If you don't see the most recent version of their files, try git pull origin <your-teammate's-branch-name>
Run your server on their branch and make sure everything looks okay
Once server is running, use your browser to navigate to api/products to ensure that correct information is displayed.

Afterwards, in Visual Studio, go to the TestProduct.cs file under TestBangazonAPI and right-click on it and click on Run Tests. Ensure that all tests come back green.

Check verbiage and comments to make sure they are legible and understandable.

**Review by ColonelSped — APPROVED** [review]

**Comment by ColonelSped** [comment]
Please merge

**Pull request merged**